compiler/noirc_frontend/src/hir/comptime/interpreter.rs

```diff
@@ -1236,11 +1247,9 @@ impl<'local, 'interner> Interpreter<'local, 'interner> {
                 let mut result = self.call_function(function_id, arguments, bindings, location)?;
                 if call.is_macro_call {
                     let expr = result.into_expression(self.elaborator.interner, location)?;
-                    let expr = self
-                        .elaborator
-                        .elaborate_item_from_comptime(self.current_function, |elab| {
-                            elab.elaborate_expression(expr).0
-                        });
+                    let expr = self.elaborate_item(self.current_function, |elab| {
```

Comment:
Please add `elab` to `cspell` or else use another word

Reply:
Renamed